site/docs/v1/tech/identity-providers/index.adoc

```diff
@@ -76,4 +95,143 @@ An email address or domain may be provided in the `login_hint` request parameter
 
 You can read more about the `login_hint` and `idp_hint` parameters in the link:/docs/v1/tech/oauth/endpoints/[OAuth Endpoints documentation].
 
-include::docs/v1/tech/identity-providers/_linking-strategies.adoc[]
+== Linking Strategies
+
+image::identity-providers/linking-strategy.png[Linking Strategies, role=top-cropped bottom-cropped, width=1200]
+
+The linking strategy to use when creating the link between the Identity Provider and the user.
+
+The possible values are:
+
+* `Create a Pending Link` - Do not automatically link, instead return a pending link identifier that can be used to link to an existing user.
+* `Anonymously Link` - Always create a link based upon the unique Id returned by the identity provider. A username or email is not required and will not be used to link the user. A reconcile lambda will not be used in this configuration.
+* `Link On Email` - Link to an existing user based upon email. A user will be created with the email returned by the identity provider if one does not already exist.
+* `Link On Email. Do not create the user if they do not exist` - Only link to an existing user based upon email. A user will not be created if one does not already exist with email returned by the identity provider.
+* `Link On Username` - Link to an existing user based upon username. A user will be created with the username returned by the identity provider if one does not already exist.
+* `Link On Username. Do not create the user if they do not exist` - Only link to an existing user based upon username. A user will not be created if one does not already exist with the username returned by the identity provider.
+
+Here's a table illustrating the alternatives:
+
+[cols="1,1,1,3"]
+|===
+| Strategy | User must exist in FusionAuth | User linked on | Use when the identity provider
+
+| Create a Pending Link | Yes | Yes, user must login to existing account | Has a different email or username than an existing FusionAuth identity and users know enough to link them.
```

Comment:
I don't think that they have to exist within FusinAuth.  Our OAuth flow presents this flow
![Screen Shot 2021-10-04 at 4 27 23 PM](https://user-images.githubusercontent.com/16090626/135933783-a4fd886a-f33c-4837-a8e1-b1f21d477e1b.png)

So you can create a user

Reply:
It was because self service registration was enabled for the application.